examples/seq2seq/seq2seq_trainer.py

```diff
@@ -41,12 +41,21 @@
 
 
 class Seq2SeqTrainer(Trainer):
-    def __init__(self, config, data_args, *args, **kwargs):
+    def __init__(self, config=None, data_args=None, *args, **kwargs):
```

Comment:
Make those variables optional to align better with `Trainer` and to keep 100% backwards compatibility